tldr/cross_file_calls.py

```
@@ -1557,6 +1567,100 @@ def _parse_elixir_import_node(node, source: bytes) -> dict | None:
+        if node.type == "variable_declaration":
+            _parse_zig_import_decl(node, source, imports)
+        elif node.type == "usingnamespace_expression":
+            _parse_zig_usingnamespace(node, source, imports)
+        for child in node.children:
+            walk_tree(child)
+
+    walk_tree(tree.root_node)
+    return imports
+
+
+def _get_zig_parser() -> Any:
```

Ruff (0.14.14)

Dynamically typed expressions (typing.Any) are disallowed in `_get_zig_parser`

(ANN401)